diff --git a/node_modules/nodejslack/index.js b/node_modules/nodejslack/index.js
index 245328f..51c417e 100755
--- a/node_modules/nodejslack/index.js
+++ b/node_modules/nodejslack/index.js
@@ -640,9 +640,13 @@ Slack.prototype._delete = function(endpoint, data) {
 Slack.prototype._request = function(httpFunction, endpoint, data) {
 
     var requestData = {
-        url: 'https://slack.com/api/' + endpoint+'?token='+this.token,
+        headers: {
+            'Authorization': 'Bearer ' + this.token
+        },
+        url: 'https://slack.com/api/' + endpoint,
         formData: data
     };
+    console.log("hey")
 
     return httpFunction(requestData)
     .then(function (response) {
