diff --git a/node_modules/appcenter-cli/dist/commands/codepush/lib/react-native-utils.js b/node_modules/appcenter-cli/dist/commands/codepush/lib/react-native-utils.js
index 1e67a28..ff4c3f8 100644
--- a/node_modules/appcenter-cli/dist/commands/codepush/lib/react-native-utils.js
+++ b/node_modules/appcenter-cli/dist/commands/codepush/lib/react-native-utils.js
@@ -448,7 +448,7 @@ function getHermesCommand() {
     if (fileExists(hermesEngine)) {
         return hermesEngine;
     }
-    return path.join("node_modules", "hermesvm", getHermesOSBin(), "hermes");
+    return path.join("node_modules", "react-native", "node_modules", "hermes-engine", getHermesOSBin(), "hermesc");
 }
 function getComposeSourceMapsPath() {
     // detect if compose-source-maps.js script exists
