diff --git a/node_modules/react-native-reanimated/ios/native/REAIOSScheduler.mm b/node_modules/react-native-reanimated/ios/native/REAIOSScheduler.mm
index 793dd7e..5525fc2 100644
--- a/node_modules/react-native-reanimated/ios/native/REAIOSScheduler.mm
+++ b/node_modules/react-native-reanimated/ios/native/REAIOSScheduler.mm
@@ -10,6 +10,9 @@
 }
 
 void REAIOSScheduler::scheduleOnUI(std::function<void()> job) {
+  if (module.lock() == nullptr) {
+     return;
+  }
   if([NSThread isMainThread]) {
     if (module.lock()) job();
     return;
@@ -20,8 +23,9 @@
     if (module.lock()) triggerUI();
     return;
   }
+  __block std::weak_ptr<NativeReanimatedModule> blockModule = module;
   dispatch_async(dispatch_get_main_queue(), ^{
-    if (module.lock()) triggerUI();
+     if (blockModule.lock()) triggerUI();
   });
 }
 
